tasks/config/sample.ts

```diff
@@ -0,0 +1,36 @@
+import Config from '../../lib/Config';
+import { isPlainObject } from '../../lib/utils/utils';
+import fs from 'fs';
+import path from 'path';
+
+export default async () => {
+  let result = '';
+
+  const recursivelyConvertJsonToToml = (json: any, prefix: string) => {
+    const nestedPairs: any = [];
+    const simplePairs: any = [];
+
+    if (!json) return;
+
+    Object.keys(json).sort().forEach((key: any) => {
+      if (Object.prototype.toString.call(json[key]) !== '[object Function]') {
+        (isPlainObject(json[key]) ? nestedPairs : simplePairs).push([key, json[key]]);
+      }
+    });
+
+    if (!(prefix === '' || simplePairs.length === 0)) {
+      result += '[' + prefix + ']\n';
```

Comment:
I believe here is where we'd want to add a blank line (per my other comment above)?

Reply:
indeed